docs/source/02_Terminology.md

```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
 | `ascii` | utf-8 codes 0 to 127 (single byte) |
 | `nonascii` | utf-8 codes greater than 128 (multi-byte) |
 | `printable` | ASCII 32 <= code < 127 |
-|`lowercase` | ASCII characters a-z |  
+| `lowercase` | ASCII characters a-z |  
 | `uppercase` | ASCII characters A-Z |
 | `letters` | `lowercase` and/or `uppercase` |
 | `text` | `printable` and/or `nonascii` |
 | `digits` | 0-9 |
 | `tab` | ASCII code 09 |
 | `newline` | ASCII code 10 (linefeed) |
 | `blank` | ASCII code 32 |
+| `exclamation` | ASCII code 33 |
+| `double-quote ` | ASCII code 34 |
```

Comment:
Remove extra space between "quote" and backtick.

Reply:
Corrected #574

Reply:
resolved #574